Cargo.toml

```diff
@@ -119,15 +119,16 @@ prost = { version = "0.12" }
 icelake = { git = "https://github.com/icelake-io/icelake", rev = "5cdcdffd24f4624a0a43f92c5f368988169a799b", features = [
   "prometheus",
 ] }
-arrow-array = "49"
-arrow-arith = "49"
-arrow-cast = "49"
-arrow-schema = "49"
-arrow-buffer = "49"
-arrow-flight = "49"
-arrow-select = "49"
-arrow-ord = "49"
-arrow-row = "49"
+arrow-array = "48.0.1"
```

Comment:
Yes. If we use 49. Cargo will reload a 48 on the dependent link, and then the output of our `to_record_batch_with_schema` method is 49. It cannot be input into the write of delta because the parameter accepted by write is 48.

Reply:
See https://github.com/risingwavelabs/risingwave/pull/13600#issuecomment-1835418339.